src/core/public/http/fetch.test.ts

```diff
@@ -481,6 +481,21 @@ describe('Fetch', () => {
 
       expect(ndjson).toEqual(content);
     });
+
+    it('should make requests for text/event-stream content', async () => {
+      const body = 'data: ';
+      fetchMock.post('*', () => ({
+        body,
+        headers: { 'Content-Type': 'text/event-stream' },
```

Comment:
can we add a case which headers is `{ 'Content-Type': 'text/event-stream;charset=UTF-8'},`

Reply:
Sure, added.